interface/usergroup/adminacl.php

```diff
@@ -28,9 +28,7 @@
   ?>
 <html>
   <head>
-    <?php
-      call_required_libraries(array('jquery-min-3-1-1'));
```

Comment:
@MigDinny I don't see why you cannot use jquery-min-1-2-1. By removing this am I correct in the fact that the only change you made was to reformat the code?

Reply:
If it works with 311, then I see no issue with proceeding towards dumping the old libs.  Ja?  Less code is less code, and with various recent hard-to-patch security flaws in things (admittedly an issue for clients more than servers) we have good reason to tighten our overview of 3rd party assets.